run_tests.py

```diff
@@ -148,12 +148,13 @@ def get_dir(default):
         out = op.normpath(op.join(sfepy.data_dir, default))
     return out
 
-usage = """%prog [options] [test_filename[ test_filename ...]]"""
+usage = """%(prog)s [options] [test_filename[ test_filename ...]]
+Use ./%(prog)s -h to get detailed help."""
```

Comment:
It is not necessary here - the script runs all the tests without arguments, and -h prints this info anyway.